exercises/practice/forth/.meta/example.lua

```diff
@@ -0,0 +1,70 @@
+local function eval_definition(definition, user_words)
+  for name, body in definition:gmatch(': (%S+)%s+(.+)') do
+    assert(not tonumber(name))
+    user_words[name:lower()] = body:gsub('%S+', function(s)
+      return user_words[s] or s
+    end)
+  end
+end
+
+local function pop(stack)
+  assert(#stack > 0)
+  return table.remove(stack)
+end
+
+local function push(stack, element)
+  assert(element ~= nil)
+  table.insert(stack, element)
+end
+
+local function eval_code(code, stack, user_words)
+  for element in code:lower():gmatch('%S+') do
+    if user_words[element] then
+      eval_code(user_words[element], stack, user_words)
+    else
+      if element == '+' then
+        push(stack, pop(stack) + pop(stack))
+      elseif element == '-' then
+        push(stack, -pop(stack) + pop(stack))
+      elseif element == '*' then
+        push(stack, pop(stack) * pop(stack))
+      elseif element == '/' then
+        local divisor = pop(stack)
+        local dividend = pop(stack)
+        assert(divisor ~= 0)
+        push(stack, math.floor(dividend / divisor))
```

Comment:
```suggestion
        push(stack, dividend // divisor)
```
Integer division?

Reply:
Yep, great suggestion. I've spent too much time in older Luas.